### ISSUE TITLE
Initial textarea value not shown on model edit

### ISSUE THREAD

**Issue by Eoler** [rainlab/translate-plugin#135]
The problem is with TranslatableModel behavior and textarea backend form widget on edge build 342 / Rainlab.Translate 1.1.4. Two languages defined, default is not English.

Tested on oc-test-plugin / Review model - added lines:
```
    public $implement = ['@RainLab.Translate.Behaviors.TranslatableModel'];
    /**
     * @var array Attributes that support translation, if available.
     */
    public $translatable = ['content'];
```
Opening second test record for editing: /backend/october/test/reviews/update/2
initial value "Excellent design work" is not populated in textarea form field.

**Comment by Eoler** [comment]
Fixed by #125, please merge.

**Comment by side-by-side** [comment]
Corrected pull request is #137. Waiting for merge...

**Comment by Eoler** [comment]
Fixed in 1.1.7
